gcapi/check_version.py

```diff
@@ -0,0 +1,49 @@
+import warnings
+from importlib.metadata import version as get_version
+
+import httpx
+from packaging import version
+
+
+class UnsupportedVersionError(Exception):
+    pass
+
+
+def check_version(base_url):
+    package_name = "gcapi"
+    try:
+        current_version = get_version(package_name)
+        with httpx.Client() as client:
+            response = client.get(f"https://pypi.org/pypi/{package_name}/json")
+            latest_version = response.json()["info"]["version"]
```

Comment:
Now that we have the endpoint we should put this information into our endpoint and save both a call and a dependency on pypi being up.

Reply:
Good point, how would you handle the versioning on the GC side though? Periodic pull from pypi or manual?

Reply:
It can be manual for now, taken from an environment variable, less dependencies on external systems the better. Don't think it's too much extra work given the frequency of releases. Then we could automate it later in terraform if needed.